earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -74,7 +80,18 @@ gdal_extract = gdal_env.Command(
         'tar xzf %s\n'
         'touch %s'% (current_dir, current_dir, gdal_source, gdal_target))])
 
-# [2] Extract Kakadu
+
+# [2] Extract openjpeg
+gdal_target = '%s/.extract_openjpeg' % current_dir
+openjpeg_extract = gdal_env.Command(
+    gdal_target, [openjpeg_source, gdal_extract],
+    [gdal_env.MultiCommand(
+        'cd %s\n'
+        'tar xzf %s\n'
+        'touch %s' % (build_root, openjpeg_source, gdal_target))])
+
+
+# [3] Extract Kakadu
```

Comment:
Should this be kept (commented out) or just removed? Do we ever plan to support it properly?